pkg/rewards/show.go

```diff
@@ -203,12 +206,20 @@ func handleRewardsOutput(
 		if err != nil {
 			return err
 		}
-		if outputFile != "" {
-			return common.WriteToFile(out, outputFile)
+		if cfg.Output != "" {
+			return common.WriteToFile(out, cfg.Output)
 		} else {
 			fmt.Println(string(out))
 		}
 	} else {
+		fmt.Println()
+		if cfg.ClaimTimestamp == "latest" {
```

Comment:
consider putting latest and latest_active in a constant

Reply:
good call, I am using it at multiple places. changing now